decidim-core/app/models/decidim/user_suspension.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Decidim
+  class UserSuspension < ApplicationRecord
+    MINIMUM_JUSTIFICATION_LENGTH = 10
+
+    belongs_to :user, class_name: "Decidim::User", foreign_key: :decidim_user_id
+    belongs_to :suspending_user, class_name: "Decidim::User"
+
+    # validates :justification, presence: true, length: { minimum: MINIMUM_JUSTIFICATION_LENGTH }
```

Comment:
If it's not used better remove, but was it intentionally commented ?
It seems the validation is required no?

Reply:
There is another PR opened for things mentioned in the original request (in comments).
https://github.com/decidim/decidim/pull/6941
There is also included the part with renaming everywhere in the interface  with "blocked"

Reply:
SuspendUserForm has already a validation for "justification". I do not think that will be needed here also, so I will delete it.